cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -17,13 +15,8 @@
 ]
```

Comment:
**Optional**
Looks like all the tests use the same `virtualcenter_provider`, consider adding the fixture mark here and removing from functions.